start

```diff
@@ -401,14 +416,25 @@ function init_soroban_rpc() {
     echo "soroban rpc: already initialized"
     return 0
   fi
+
+  SOROBAN_RPC_CAPTIVE_CORE_CFG=$COREHOME/etc/soroban-rpc-captive-core.cfg
+  cp $COREHOME/etc/stellar-captive-core.cfg $SOROBAN_RPC_CAPTIVE_CORE_CFG 2>/dev/null || true
+  run_silent "finalize-rpc-captivecore-db" sed -ri "s*__DATABASE__*sqlite3://$COREHOME/stellar-rpc.db*g" $SOROBAN_RPC_CAPTIVE_CORE_CFG
+  run_silent "finalize-rpc-captivecore-peer-port" sed -ri "s*__PEER_PORT__*11726*g" $SOROBAN_RPC_CAPTIVE_CORE_CFG
```

Comment:
i'm looking into peer port now, since core and captive core instances running on same host they can't all be opening listeners on same network interface and port 11625

Reply:
found the issue, it wasn't peer port related, I had to include setting `RUN_STANDALONE=false` on core cfg when captive core is activated either from horizon or rpc.